src/frontend/src/lib/components/transactions/TransactionsDateGroup.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import { SLIDE_DURATION } from '$lib/constants/transition.constants';
+	import type { AllTransactionsUi } from '$lib/types/transaction';
+
+	export let date: string;
+	export let transactions: AllTransactionsUi;
+</script>
+
+{#if transactions.length > 0}
```

Comment:
An empty array can happen?

Reply:
it shouldn't according to its usage, but it is just to have a failsafe and not showing the date if for some reason the list is empty. I can remove it if you want

Reply:
good one! yes! will do it

Reply:
Done with:

- PR #3601